src/component-library/Modal/Modal.tsx

```diff
@@ -14,14 +14,15 @@ import { ModalContext } from './ModalContext';
 
 type Props = {
   children: ReactNode;
+  isCentered?: boolean;
```

Comment:
@tomjeatt what should be the default value? The modal in the center of the screen or on the top?

Reply:
Thank you Tom